templates/schema.json

```diff
@@ -11,6 +11,10 @@
       "type": "string",
       "description": "Longer description of the template."
     },
+    "category": {
```

Comment:
Alternate idea from github-land: "labels"

Reply:
to me labels suggests an array not a string, which I am open to but would add some other complications to my original implementation of having things in at most one bucket on the list.

Reply:
Oh yeah definitely labels would be an array.

Reply:
okay i switched from a category string to a label array.

Once we merge it you can start using it when thinking through the template list view UI Jordan